update_aimpoint_data.py

```diff
@@ -76,14 +91,16 @@ def add_asol_to_h5(filename, asol):
 
 # Define file names
 h5_file = os.path.join(opt.data_root, 'aimpoint_asol_values.h5')
+
+# When we go to PY3, just remove '.shelve' to make this work.
 obsid_file = os.path.join(opt.data_root, 'aimpoint_obsid_index.shelve')
 
 # Get obsids in date range
-db = DBI(dbi='sqlite', server='/data/aca/archive/obspar/archfiles.db3')
-obs = db.fetchall('select obsid, tstart from archfiles where tstart > {}'
-                  ' and tstart < {}'
-                  .format(start.secs, stop.secs))
-db.conn.close()
+mica_obspar_db = os.path.join(MICA_ARCHIVE, 'obspar', 'archfiles.db3')
```

Comment:
I figured it was not unreasonable to update this path to use MICA_ARCHIVE, though really it looks like this code is just compensating for a missing method in `mica` and we could update mica to help with that.